src/org/usfirst/frc/team1086/CameraCalculator/Sighting.java

```diff
@@ -46,7 +47,26 @@ public Sighting(MatOfPoint m){
         centerX = x + width / 2;
         centerY = y + height / 2;
     }
-    
+    /**
+     * Manual Sighting creation without using OpenCV
+     * @param x the center x of the sighting
+     * @param y the center y of the sighting
+     * @param width width of the sighting
+     * @param height the height of the sighting
+     */
+    public Sighting(int signature, int x, int y, int width, int height){
+		//NOTE: ADD POINTS A
+    	this.centerX=x;
+		this.centerY=y;
+		this.x=(int) (centerX-width/2);
+		this.x=(int) (centerY+height/2);
```

Comment:
Should be `this.y`

😉

Reply:
Grant: I was a bit unclear in what I wanted from this, but you got the hard part down: communication from RIO and Pixy. I'm going to take that and work it in to my code rather than work my pre-existing code to fit it.